autoupdate/main.go

```diff
@@ -147,6 +160,8 @@ func update(log *zap.Logger) int {
 	defaultDownloadPath := flag.String("download", "", "default download path")
 	// Input Where your local node is running on, default value is localhost:5001.
 	url := flag.String("url", "localhost:5001", "your node's http server addr")
+	statusServerDomain := flag.String("ssd", "https://test.db.btfs.io", "the status server domain")
```

Comment:
If you have these domain in configuration, please read from config.

Reply:
Because autoupdate is a separate binary, we may not want to import dependency from config repo, so you can ignore my last comment.